salt/modules/rbenv.py

```diff
@@ -380,7 +380,12 @@ def do(cmdline, runas=None, env=None):
     if not env:
         env = {}
 
-    env['PATH'] = '{0}/shims:{1}'.format(path, os.environ['PATH'])
+    env[str('PATH')] = salt.utils.stringutils.to_str(   # future lint: disable=blacklisted-function
+        '{0}/shims:{1}'.format(
+            path,
+            salt.utils.stringutils.to_unicode(os.environ['PATH'])
+        )
+    )
```

Comment:
NOTE: I know it may seem weird that this code converts the PATH env var to unicode and then back to a `str` type. The unicode conversion is to ensure that the `.format()` doesn't raise a decode error, and the `to_str()` ensures the result is a `str` type like we need it to be.

Reply:
Maybe you should add a comment above explaining this, just so someone doesn't see it in the code and remove it in the future.